sdk/go/pulumi/types_test.go

```diff
@@ -352,7 +357,7 @@ func TestToOutputAnyDeps(t *testing.T) {
 	v, known, secret, deps, err := await(out)
 	assert.True(t, known)
 	assert.False(t, secret)
-	assert.ElementsMatch(t, []Resource{stringDep1, stringDep2, intDep1, intDep2, boolDep1, boolDep2, res}, deps)
+	assert.ElementsMatch(t, []Resource{stringDep1, stringDep2, intDep1, intDep2, boolDep1, boolDep2}, deps)
```

Comment:
Why doesn't this depend on res anymore?

Reply:
The dependency on `res` is inconsistent with the behavior of `All` in other languages, which do not include resource instances encountered during the crawl in the list of deps for the output.